### PR TITLE
fix: Added the Open Food facts logo, changed the button to the orange theme and fixed the broken link. 

### PR DESCRIPTION
**Pull request by Aarthy153**
…e and corrected the play store broken link.

### What
Added the Open Food facts logo, changed the button to the orange theme, and fixed the broken link. 

### Screenshot

image

### Fixes bug(s)
#28

### CODE REVIEW

**Comment by Aarthy153** [comment]
@teolemon Requesting your review for the PR

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <body>
         <div class="navbar">
             <div class="navbar-container">
-                <a href="#"><img src="img/logo_nav@3x.png" alt="Open Food Facts" width="150px"></a>
+                <a href="#"><img src="img/open food facts logo.png" alt="Open Food Facts" width="250px"></a>
```
Comment (teolemon) [inline index.html]: Can you normalize the name?

**Review by teolemon — APPROVED** [review]

**Comment by teolemon** [comment]
Looks good to me @Aarthy153 👍 can you just normalize the file name?
Also please note that this project is not currently deployed to production, as it looks a little bit out of sync with the new graphic charter

**Comment by Aarthy153** [comment]
@teolemon Can I develop this prototype to match the official theme and colour codes of the Open food facts? So then it will suitable for the deployment right?

File: index.html (hunk 1 above)
Comment (Aarthy153) [reply index.html]: Sure @teolemon

**Comment by Aarthy153** [comment]
@teolemon I have done the required changes in the file name.

**Comment by M123-dev** [comment]
The current "mobile app" website is this one https://world.openfoodfacts.org/open-food-facts-mobile-app

In terms of design we can surely use the new one, but from a technical perspective that's a different thing, 
Feel free to create a issue for that in https://github.com/openfoodfacts/openfoodfacts-server

**Comment by teolemon** [comment]
I'm going to merge anyway, it will fix current issues. Thanks @Aarthy153

**Pull request merged**